utils/korath-cipher.py

```diff
@@ -0,0 +1,142 @@
+#! /usr/bin/env python
+
+# Copyright (c) 2023 by an anonymous author
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+# This script implements the cipher by Lia Gerty described here:
+#
+#    https://github.com/endless-sky/endless-sky/pull/7101
+#
+# Works with Python 2.7 or newer.
+
+
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+#
+# INSTRUCTIONS
+#
+# Before you run this script, first you need to translate your desired
+# phrase from English to Indonesian. Here's an example:
+#
+# English: Within truth, I see only lies.
+# Indonesian: Dalam kebenaran, saya hanya melihat kebohongan.
+#
+# Next, send that Indonesian text into this script's stdin, and you'll see:
+#
+# ORIGINAL: Dalam kebenaran, saya hanya melihat kebohongan.
+# REVERSED: malaD ,naranebek ayas aynah tahilem .nagnohobek
+# EXILE:    famaS ,ralarehet aga' agrap kapimef .ranrupuhet
+# EFRETI:   fanaT ,raparebes adam adrah kahinef .ralruhubes
+#
+# Clearly, you'll need to manually correct a few things. The commas
+# and periods are in the wrong place, the wrong letters are
+# capitalized, and it's probably best to avoid common names and
+# religious concepts. (Adam = the first man in some religions.) You
+# should also make sure the end product is pronounceable, with
+# particular attention to ' characters in Exile words. These are meant
+# to represent glottal stops; if you are unfamiliar, imagine what the
+# word would sound like with a k in place of the '. If it is too
+# difficult, add some vowels or move letters around. After manual
+# tweaking, you might end up with something like this:
+#
+# EXILE:  Famas ralarehet, aga' agrap kapimef ranrupuhet.
+# EFRETI: Fanat raparebes, adas adrah kahinef ralruhubes.
+#
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+#
+# GENERAL TRANSLATION NOTES
+#
+# There are a few words with standard translations you'll need to apply manually:
+#
+# human => humanika                             (both Exile and Efreti)
```

Comment:
If anything, I would guess `humanika` is the *Quarg* word for `human`.
In the text we get upon first landing on the Efreti planets/station, they say `humani efreti`.
Speaking of which, `efreti` should probalby be in this list of direct translations.

Reply:
After discussion in another thread we determined the translation was wrong. I never updated it in this txt:

humani = human (singular)
humanika = humans (plural)

The commonality between the three languages comes from the fact that both Korath factions found out about humans initially from the Quarg.